src/main/java/rx/Completable.java

```diff
@@ -1114,6 +1114,21 @@ public final Completable compose(CompletableTransformer transformer) {
         requireNonNull(next);
         return next.delaySubscription(toObservable());
     }
+
+    /**
+     * Returns a Single which will subscribe to this Completable and once that is completed then
+     * will subscribe to the {@code next} Single. An error event from this Completable will be
+     * propagated to the downstream subscriber and will result in skipping the subscription of the
+     * Observable.
+     *
+     * @param next the Single to subscribe after this Completable is completed, not null
+     * @return Single that composes this Completable and next
+     * @throws NullPointerException if next is null
```

Comment:
nit: probably will be better to write `NullPointerException if Single is null`, just because `next` is kind of keyword in RxJava for `onNext` notifications, though Completable doesn't have it

Or you can just remove `@throws` because javadoc for `@param` already says that it shouldn't be null